src/issuance.rs

```diff
@@ -553,6 +546,27 @@ pub enum Error {
 
 impl std::error::Error for Error {}
 
+impl PartialEq for Error {
+    fn eq(&self, other: &Self) -> bool {
+        match (self, other) {
+            (IssueActionNotFound, IssueActionNotFound) => true,
+            (IssueBundleIkMismatchAssetBase, IssueBundleIkMismatchAssetBase) => true,
+            (WrongAssetDescSize, WrongAssetDescSize) => true,
+            (IssueActionWithoutNoteNotFinalized, IssueActionWithoutNoteNotFinalized) => true,
+            (AssetBaseCannotBeIdentityPoint, AssetBaseCannotBeIdentityPoint) => true,
+            (IssueBundleInvalidSignature(_), IssueBundleInvalidSignature(_)) => true, //might want to do better here?
+            (
+                IssueActionPreviouslyFinalizedAssetBase(x),
+                IssueActionPreviouslyFinalizedAssetBase(y),
+            ) => x == y,
+            (ValueSumOverflow, ValueSumOverflow) => true,
+            _ => false,
+        }
+    }
+}
+
+impl Eq for Error {}
```

Comment:
`schnorr::Error` does not derive `PartialEq` and `Eq`, so I added that. As mentioned in line 557, this could benefit from improvement.

Reply:
Removed this and moved it back inside the `derive`.